cypress/integration/Auth/LoginCypressObject.ts

```diff
@@ -0,0 +1,31 @@
+const BASE_URL = process.env.BASE_URL || `http://localhost:${process.env.PORT || 8000}`;
```

Comment:
move this file elsewhere as it is being called as test 

https://github.com/EscolaLMS/Admin/actions/runs/1516827587 see videos